frontend/javascripts/test/helpers/apiHelpers.ts

```diff
@@ -193,6 +194,10 @@ export function __setupOxalis(
     );
   Request.receiveJSON.returns(Promise.resolve({}));
   Request.sendJSONReceiveJSON.returns(Promise.resolve({}));
+  Request.sendJSONReceiveJSON
+    .withArgs(sinon.match((arg) => arg == `/api/users/${dummyUser.id}/taskTypeId`))
```

Comment:
This mock relates to the `updateLastTaskTypeIdOfUser` api. There's a saga which updates the last active task type that wasn't run before, because there was no active user. That's why I had to mock it, too. I added a comment to explain to what the route relates.